src/app/loadout-builder/process/mappers.ts

```diff
@@ -159,7 +159,11 @@ export function mapDimItemToProcessItem(
     energy: energy
       ? {
           type: energyType ?? energy.energyType,
+          // used to find best mod assignments
+          originalEnergyType: energy.energyType,
           capacity: upgradeSpendTierToMaxEnergy(defs, upgradeSpendTier, dimItem),
+          // used to find best mod assignments
+          originalCapacity: energy.energyCapacity,
```

Comment:
I don't like that I am adding these for someone that is never actually run in process. At some point I think it's going to be worth having two copies of the algorithm that loops over all mod permutations to check validity, one for process items and one for normal. It will be much more viable to do so when I get rid of legacy mods as they make things hard.